packages/core/src/components/menu/menuDivider.tsx

```diff
@@ -45,16 +45,11 @@ export class MenuDivider extends React.Component<MenuDividerProps> {
 
     public render() {
         const { className, title, titleId } = this.props;
-        if (title == null) {
-            // simple divider
```

Comment:
I liked the previous code style, even though it was more verbose. I like that it explicitly lists out the two types of dividers. Can you refactor your PR to make the most minimal change necessary?